packages/babel-generator/src/generators/modules.js

```diff
@@ -45,12 +45,6 @@ export function ExportAllDeclaration(node: Object) {
   this.word("export");
   this.space();
   this.token("*");
-  if (node.exported) {
```

Comment:
Ah yeah I was thinking of `export * as ns from 'mod';` from https://babeljs.io/docs/plugins/transform-export-extensions/ but it's a different node - ExportNamespaceSpecifier